generator/AWSPSGeneratorLib/HelpMaterials/Examples/LM/Remove-LMPermission.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$json =  Get-LMPolicy -FunctionName "MylambdaFunction123"| Select-Object -ExpandProperty  Policy | ConvertFrom-Json
```

Comment:
It would be nice to Add PS C:\\> before `Remove-LMPermission` - since we want to indicate, they are still commands and not output. 

Also, you could optimize
`$policyStatements = Get-LMPolicy restart-service -Select Policy | ConvertFrom-Json | Select-Object -ExpandProperty statement`

Reply:
Simplified to below :
$json =  Get-LMPolicy -FunctionName "MylambdaFunction123" -Select Policy | ConvertFrom-Json| Select-Object -ExpandProperty Statement
C:\> Remove-LMPermission -FunctionName "MylambdaFunction123" -StatementId $json[0].Sid